ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-community-lists.j2

```diff
@@ -0,0 +1,15 @@
+{# eos - ip-community-lists #}
+{% if ip_community_lists is arista.avd.defined %}
+!
+{%     for community_list in ip_community_lists %}
+{%         for entry in community_list.entries %}
```

Comment:
Need to add a defined test for **community_list.entries** above line5. Alternatively we can use ```for entry in community_list.entries | arista.avd.default([])```